src/Features/Hud/Hud.cpp

```diff
@@ -858,6 +859,14 @@ HUD_ELEMENT_MODE2(ent_slot_serial, "0", 0, 4096,
 	ctx->DrawElement("ent slot %d serial: %d", mode, serial);
 }
 
+HUD_ELEMENT_MODE2(fps, "0", 0, 2, "Show fps (frames per second) on the SAR hud\n", HudType_InGame | HudType_Paused) {
```

Comment:
any reason not to allow this in the menu too?

Reply:
personally thought it wouldn't be too useful, added it now though